go/vt/topo/keyspace.go

```diff
@@ -34,7 +38,20 @@ import (
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
-// This file contains keyspace utility functions
+// This file contains keyspace utility functions.
+
+// Default concurrency to use in order to avoid overhwelming the topo server.
+var DefaultConcurrency int64
```

Comment:
Ugh, I just realized the flag value was always int64, and there's no good reason for that. Should we change this to int? Is there any reason not to?

Reply:
No reason not to. `int` is `int64` on 64-bit platforms, and for 32-bit they can't really be using bigger numbers anyway 🤷

Reply:
32 bit machines support 64 bit integers.

Reply:
You can use int64 on 32 bit machines -- two 32 bit pieces are used. We (the royal we) were using long long / int64 when needed when most machines were still 32 bit architectures.

Reply:
I'm fine moving this to int though. I initially started there but then preferred not to change the existing flag.

Reply:
Changed it here (and now using it in another location where we should be): https://github.com/vitessio/vitess/pull/15047/commits/99a7f86fbee1f6e4fa226984e93d20f80ee978ec